CMakeLists.txt

```diff
@@ -524,23 +516,12 @@ add_custom_target(link_target ALL
                   build/compile_commands.json ../compile_commands.json)
 endif()
 
-add_custom_command(TARGET foedag-bin POST_BUILD
```

Comment:
Why so many deletions here ?

Reply:
I made this more simple - copy whole etc folder instead of copy every folder from etc.

Reply:
Ok, did you ensure this is doesn't break foedag_rs and raptor builds ?

Reply:
yes